entropy/windows/Raaz/Entropy.hsc

```diff
@@ -30,18 +31,19 @@ type HCRYPTPROV = Ptr ()
 foreign import WINDOWS_CCONV unsafe "Wincrypt.h CryptGenRandom"
     c_CryptGenRandom :: HCRYPTPROV -> Word32 -> Ptr Word8 -> IO Bool
 
-foreign import WINDOWS_CCONV unsafe "Wincrypt.h CryptAcquireContext"
+foreign import WINDOWS_CCONV unsafe "Wincrypt.h CryptAcquireContextW"
     c_CryptAcquireContext :: Ptr HCRYPTPROV -> CWString -> CWString
                           -> Word32 -> Word32 -> IO Bool
 
 foreign import WINDOWS_CCONV unsafe "Wincrypt.h CryptReleaseContext"
     c_CryptReleaseContext :: HCRYPTPROV -> Word32 -> IO Bool
 
 -- | Cache the crypto context so we don't have to create it on each call.
-cryptoContext :: ForeignPtr (HCRYPTPROV)
+cryptoContext :: ForeignPtr HCRYPTPROV
```

Comment:
Also you may disable the -v in the appveyor builds so that we can get to the error quickly.
See appveyor.yml